### DISCUSSION TITLE
On the localhost and on the readthedocs server, the code display is different

### DISCUSSION THREAD

**Discussion by HeaTTheatR** [pradyunsg/furo#480] [Potential Issue] [answered]
It is quite possible that my crooked hands are to blame for everything... On the localhost (when I collect documentation locally), the code is displayed according to the `Furo` theme:

![Снимок экрана 2022-07-12 в 22 26 19](https://user-images.githubusercontent.com/16930280/178580019-82314a45-734c-4cbd-94dd-44a1c3b1ca64.png)

But on the `readthedocs` server (after the automatic `workflows` build), the code display is different:

![Снимок экрана 2022-07-12 в 22 26 25](https://user-images.githubusercontent.com/16930280/178580574-134911e1-7ef2-4322-a8bc-a1802fa6bc0f.png)

In the `conf.py` file just changed the theme name:

```python
[...]

html_theme = "furo"

[...]
```

What could be the problem?

**Comment by pradyunsg** [comment]
Do you have a link for me to investigate?

**Reply by HeaTTheatR** [reply]
Sorry, I didn't quite understand which link you mean? A link to the readthedocs build log?

**Reply by pradyunsg** [reply]
I meant a link to the built documentation on readthedocs as well as the repository that these were built with.

I need to reproduce or, at least, point a browser at the page to investigate what's going on. :)

**Comment by HeaTTheatR** [comment, marked as answer]
I cleared the browser cache and everything worked as it should.

**Reply by pradyunsg** [reply]
Ah, fun.

Furo doesn't do cache-busting on the Pygments srylesheet since Sphinx adds it to the page. All the other stylesheets that Furo adds have cache-busting hashes on them.

That's likely why you saw this behavior. :)
